pkg/store/use.go

```diff
@@ -18,5 +19,14 @@ func GetUseLocation() string {
 
 // GetActiveBinaryLocation returns the path to the currently active Terraform binary.
 func GetActiveBinaryLocation() string {
-	return filepath.Join(GetUseLocation(), TerraformBinaryName)
```

Comment:
Would it be a good idea to store the result of GetTerraformBinaryName() in a variable, logging it, and then passing the variable to Join?

Reply:
While I think logging the binaryName that's being used it useful, we currently don't do any (debug) logging nor do we have debug flags added to cobra. I'd prefer to make that a separate PR.

Reply:
Agreed